### ISSUE TITLE
--simulate option does not work for push

### ISSUE THREAD

**Issue by akishin** [welaika/wordmove#480]
In push case, it seems that the --simulate option does not work because the options key becomes a String.

pull
https://github.com/welaika/wordmove/blob/master/lib/wordmove/cli.rb#L68

```
deployer = Wordmove::Deployer::Base.deployer_for(options.deep_symbolize_keys)
```

push
https://github.com/welaika/wordmove/blob/master/lib/wordmove/cli.rb#L92

```
deployer = Wordmove::Deployer::Base.deployer_for(options)
```

Is not the deep_symbolize_keys necessary also for push method?

**Comment by pioneerskies** [comment]
Hello @akishin ,

not only you are totally right, but we have already a fix in 9541d8ff59bddc7e391e60c6b9a237f2d2c9906c

The branch should be merged and thus released **really** soon.

Thank you very much

**Comment by pioneerskies** [comment]
@akishin https://github.com/welaika/wordmove/releases/tag/v3.1.2 is out with the fix :)

Thanks again
